packages/salesforcedx-apex-debugger/package.json

```diff
@@ -42,7 +42,7 @@
     "watch": "tsc -watch -p .",
     "clean":
       "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
-    "test": "npm run test:unit && npm run test:integration",
+    "test": "npm run test:unit",
```

Comment:
Why are we now exclusively running unit tests with the `test` script ?

Reply:
There were two integration tests in this package that were being `skipped,` which was stopping codecov from reporting. I was going to comment the tests out, but I realized those are the only two `integration` tests in the package so I adjusted the `test` script instead.

Reply:
Got it, can you log a child story so we can fix them later ?